symfony-app-skeleton/src/Entity/OrderShipmentEntity.php

```diff
@@ -0,0 +1,452 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\OrderShipmentEntityRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * @ORM\Entity(repositoryClass=OrderShipmentEntityRepository::class)
+ */
+class OrderShipmentEntity
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $clientUserID;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $transportationType;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $target;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $supplierID;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $distributorID;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $exportWarehouseID;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $importWarehouseID;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $quantity;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $image;
+
+    /**
+     * @Gedmo\Timestampable(on="create")
+     * @ORM\Column(type="datetime")
+     */
+    private $createdAt;
+
+    /**
+     * @Gedmo\Timestampable(on="update")
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private $updatedAt;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $productCategoryID;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $unit;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $receiverName;
+
+    /**
+     * @ORM\Column(type="string", length=50, nullable=true)
+     */
+    private $receiverPhoneNumber;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $markID;
+
+    /**
+     * @ORM\Column(type="integer", nullable=true)
+     */
+    private $packetingBy;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $paymentTime;
+
+    /**
+     * @ORM\Column(type="float", nullable=true)
+     */
+    private $weight;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
+     */
+    private $QRcode;
```

Comment:
please use normal naming system, and make search for text length that will be suit the size of qr code